src/services/notifications.service.js

```diff
@@ -79,6 +83,11 @@ const transformDataForNotify = (registration, lcContactConfig, i18n) => {
       lcInfo.local_council_phone_number_standards =
         lcContactConfig.standards.local_council_phone_number;
     }
+
+     if (lcContactConfig.standards.local_guidance_link) {
+       lcInfo.local_guidance_link_standards =
+         lcContactConfig.standards.local_guidance_link;
```

Comment:
The parameter name is different, the frontend uses local_council_guidance_link, while here it is local_guidance_link